modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -33,6 +33,7 @@
 #define MINIMP3_NO_STDIO
 
 #include "audio_stream_mp3.h"
+#include "resource_importer_mp3.h"
 
 #include "core/io/file_access.h"
```

Comment:
```suggestion

#include "core/io/file_access.h"
#include "resource_importer_mp3.h"
```
The header belonging to the source file should be separate, and the other includes grouped

Reply:
Actually it should be:
```
#include "audio_stream_mp3.h"

#include "resource_importer_mp3.h"

#include "core/io/file_access.h"
```

Following the yet-undocumented rules from https://github.com/godotengine/godot/pull/75944.

Reply:
Updated.